webpack.config.js

```diff
@@ -1,6 +1,167 @@
-const webpackConfigGenerator = require('./webpackConfigGenerator');
+const path = require('path');
+const webpack = require('webpack');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const BabiliPlugin = require('babili-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 module.exports = [
-   ...webpackConfigGenerator('webextension'),
-   ...webpackConfigGenerator('firefox')
-];
\ No newline at end of file
+  {
+    entry: './src/content_script.js',
+    output: {
+      path: path.resolve(__dirname, `./build/`),
+      publicPath: '/sys/jsoneditor/dist/img/',
+      filename: 'content_script.js',
+      library: 'starcounterDebugAid',
+      libraryTarget: 'var'
+    },
+    plugins: [
+      new CopyWebpackPlugin([
+        { from: 'icons', to: 'icons' },
+        { from: 'popup', to: 'popup' },
+        { from: 'options', to: 'options' },
+        {
+          from: 'manifest.json',
+          to: 'manifest.json'
+        }
```

Comment:
What it actually does? copy `{from: x, to: x}`?

Reply:
Yes copies the files as is. Albeit `to` is not needed here.